store/views.py

```diff
@@ -77,6 +103,45 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
-
+    
 
+    book_id = request.POST.get("bid")
+    book=BookCopy.objects.get(id=book_id)
+    if(book):
+        message="success"
+        book.status=True
+        book.borrower=None
+        book.borrow_date=None
+        book.save()
+    else:
+        message="failure"
+    response_data = {
+        'message': message,
+    }
+    return JsonResponse(response_data)
+@login_required
+@csrf_exempt
+def bookRatingView(request):
+    template_name= template_name = 'store/book_detail.html'
+    book_id = request.POST.get("bid")
+    rating_value = request.POST.get("rating")
+    book1=Book.objects.get(pk=book_id)
+    value= BookRating.objects.filter(book__exact=book1,user__exact=request.user)
+    if(value.count()>0):
+        currentuserRating= BookRating.objects.filter(book__exact=book1,user__exact=request.user).get()
+        currentuserRating.ratinguser=rating_value
+        currentuserRating.save()
```

Comment:
You've not put a backend validation on the rating, so the user can easily put invalid values of rating by editing the JS code.